scripts/staking.py

```diff
@@ -144,33 +182,7 @@ def unstake_everest(
                 )
                 input("Press Enter to continue...")
 
-            next_ts = get_next_checkpoint_ts(ledger_api, args.staking_contract_address)
-            ts_start = get_service_info(
-                ledger_api, args.service_id, args.staking_contract_address
-            )[3]
-
-            liveness_period = get_liveness_period(
-                ledger_api, args.staking_contract_address
-            )
-            last_ts = next_ts - liveness_period
-            now = time.time()
-
-            minimum_staking_duration = get_min_staking_duration(
-                ledger_api, args.staking_contract_address
-            )
-            available_rewards = get_available_rewards(
-                ledger_api, args.staking_contract_address
-            )
```

Comment:
not sure why lines 143-163 are removed

Reply:
They are not removed, they are moved above to collect this information, which is constant for all the workflow below.